### DISCUSSION TITLE
How to remove unwanted menu?

### DISCUSSION THREAD

**Discussion by stv707** [gravitational/teleport#8464] [Q&A]
Hi, I'm relatively new to teleport. I manage to install and configure it. 

My deployment is only for SSH servers, I don't need Application, Kubernetes, Databases, how do I remove this from user view (  when users log on, users should on see Servers. ) 

I Know this can be controlled via the YAML file, I manage to create roles and map users based on labels on nodes, but I cannot find any example on doc nor internet to completely remove Application, Kubernetes, Databases from user view. 

Teleport v7.2 running on Centos 7.9

**Comment by lennarttd** [comment]
Same question here. Has this been implemented?

**Comment by vasart** [comment]
I have the same question
